svn/common.py

```diff
@@ -135,6 +135,47 @@ def info(self, rel_path=None, revision=None):
         info['commit_revision'] = info['commit#revision']
 
         return info
+		
+    def get_property(self, property, rel_path=None, revision=None, recursive=False):
+        """ Return an svn property, or, if called recursively, a list of tuples associated with a
+                    relative path.
+                :param property: the property to query
```

Comment:
Let's stay away from specific parameter markup styles for now unless/until we choose a particular documentation platform. I know this is for Sphinx, and, although I like Sphinx in principle, I find the reStructuredText style grating. Since we're not documented into Sphinx at the moment, it's also not a discussion worth having yet.